torch_xla/csrc/device.cpp

```diff
@@ -78,19 +78,19 @@ torch::lazy::BackendDevice GetVirtualDevice() {
 }
 
 bool ShouldUseVirtualDevice() {
-  bool use_virtual_device =
+  bool g_use_virtual_device =
```

Comment:
This `g_use_virtual_device` variable shadows a global variable of the same name.